src/ReactDataGrid.js

```diff
@@ -102,7 +102,7 @@ const ReactDataGrid = React.createClass({
     onRowClick: React.PropTypes.func,
     onGridKeyUp: React.PropTypes.func,
     onGridKeyDown: React.PropTypes.func,
-    rowGroupRenderer: React.PropTypes.func
```

Comment:
Deleting this will break the `rowGroupRenderer`, it is injected as a prop in [BaseGrid](https://github.com/adazzle/react-data-grid/blob/0b05b5303b4d3c7e630b54a22c5571b8b60b9f6a/src/ReactDataGrid.js#L874) using spread syntax.